server/rooms.js

```diff
@@ -1471,10 +1469,10 @@ class GameRoom extends BasicChatRoom {
 		this.game = null;
 	}
 	/**
-	 * - logNum = 0    : spectator log (no exact HP)
-	 * - logNum = 1, 2 : player log (exact HP for that player)
-	 * - logNum = 3    : debug log (exact HP for all players)
-	 * @param {0 | 1 | 2 | 3} channel
+	 * - logNum = 0          : spectator log (no exact HP)
+	 * - logNum = 1, 2, 3, 4 : player log (exact HP for that player)
+	 * - logNum = 5          : debug log (exact HP for all players)
+	 * @param {0 | 1 | 2 | 3 | 4 | 5} channel
```

Comment:
Ummm, I would prefer if `|split` remained four-sided.

You should be able to see HP for your allies, anyway.

Reply:
Oh, the `sockets.js` change can be done in such a way that it passes the same message to multiple sockets. Wish I’d thought of that sooner.

Reply:
Wait, but what about FFA battles? Surely that requires six channels?

Reply:
It only does if we care enough, which I don't.

Reply:
Hmph.

Reply:
I would want a different message type if you insisted, but I really think it's not worth it in practice.

Reply:
Even if we do use the four-sided `|split`, there wouldn't be any reason for this to not be as I have it here. There still needs to be six sockets, one for each player.

Reply:
And you _know_ people will complain about this if split is four-sided in FFA battles.

Reply:
Oh, maybe I actually can just put two players on the same channel.

Reply:
This does mean rewriting every single function that ever gets passed to `Battle#add`, though.

Reply:
...which is apparently just `Pokemon#getHealth` and `Pokemon#getDetails`. Are there any more?

Reply:
No, I think that's it. Maybe also `hint`?

Reply:
Also Pressure activation in Gen 3.

Reply:
Those don't actually have anything to do with `Battle#add` they just add to the logs directly.